public/javascripts/Admin/src/Admin.GSVLabelView.js

```diff
@@ -521,12 +521,28 @@ function AdminGSVLabelView(admin, source) {
         });
     }
 
+    function _escapeHTML(str) {
+        return str.replace(/[&<>"']/g, function(match) {
+            switch (match) {
+                case '&': return '&amp;';
+                case '<': return '&lt;';
+                case '>': return '&gt;';
+                case '"': return '&quot;';
+                case "'": return '&#039;';
+                default: return match;
+            }
+        });
+    }
+
     function _handleData(labelMetadata) {
         // Pass a callback function that fills in the pano lat/lng.
         var panoCallback = function () {
             var lat = self.panorama.panorama.getPosition().lat();
             var lng = self.panorama.panorama.getPosition().lng();
-            self.modalGsvLink.html(`<a href="https://www.google.com/maps/@?api=1&map_action=pano&pano=${labelMetadata['gsv_panorama_id']}&heading=${labelMetadata['heading']}&pitch=${labelMetadata['pitch']}" target="_blank">${i18next.t('common:gsv-info.view-in-gsv')}</a>`);
+            var href = `https://www.google.com/maps/@?api=1&map_action=pano&pano=${labelMetadata['gsv_panorama_id']}&heading=${labelMetadata['heading']}&pitch=${labelMetadata['pitch']}`;
```

Comment:
The link here seems to be messed up, I think the structure of the HTML is a bit mixed up so clicking on the link doesn't do anything